app-games/src/main/java/com/aptoide/android/aptoidegames/feature_promotional/NewsBundle.kt

```diff
@@ -0,0 +1,143 @@
+package com.aptoide.android.aptoidegames.feature_promotional
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.unit.dp
+import cm.aptoide.pt.extensions.PreviewDark
+import cm.aptoide.pt.feature_editorial.domain.ArticleMeta
+import cm.aptoide.pt.feature_editorial.presentation.ArticleListUiState
+import cm.aptoide.pt.feature_editorial.presentation.ArticleListUiStateProvider
+import cm.aptoide.pt.feature_editorial.presentation.rememberEditorialListState
+import cm.aptoide.pt.feature_home.domain.Bundle
+import cm.aptoide.pt.feature_home.domain.randomBundle
+import com.aptoide.android.aptoidegames.AptoideFeatureGraphicImage
+import com.aptoide.android.aptoidegames.editorial.buildEditorialRoute
+import com.aptoide.android.aptoidegames.feature_apps.presentation.SmallEmptyView
+import com.aptoide.android.aptoidegames.home.LoadingBundleView
+import com.aptoide.android.aptoidegames.theme.AGTypography
+import com.aptoide.android.aptoidegames.theme.AptoideTheme
+import com.aptoide.android.aptoidegames.theme.Palette
+
+@Composable
+fun NewsBundle(
+  bundle: Bundle,
+  modifier: Modifier = Modifier,
+  navigate: (String) -> Unit,
+) {
+  val (uiState, reload) = rememberEditorialListState(
+    tag = bundle.tag,
+    salt = bundle.timestamp
+  )
+  NewsBundleContent(
+    uiState = uiState,
+    bundle = bundle,
+    modifier = modifier,
+    navigate = navigate
+  )
+}
+
+@Composable
+private fun NewsBundleContent(
+  uiState: ArticleListUiState,
+  bundle: Bundle,
+  modifier: Modifier = Modifier,
+  navigate: (String) -> Unit,
+) {
+  when (uiState) {
+    ArticleListUiState.Empty,
```

Comment:
In case of errors, should we show error view ? I was mapping to Unit on mine, so we don't show it